src/main/java/com/google/devtools/build/lib/exec/ExecutionOptions.java

```diff
@@ -472,6 +472,17 @@ public boolean usingLocalTestJobs() {
               + " --subcommands (for displaying subcommands in terminal output).")
   public PathFragment executionLogJsonFile;
 
+  @Option(
+      name = "execution_log_sort",
+      defaultValue = "true",
+      documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "When logging the executed spawns into files of the delimited Spawn protos, according to "
+              + " src/main/protobuf/spawn.proto, this flag can determine whether to sort log in"
+              + " a stable order or not, can be disabled to optimize memory performance.")
```

Comment:
```suggestion
          "Whether to sort the execution log. Set to false to improve memory"
              + " performance, at the cost of producing the log in nondeterministic"
              + " order.")
```

Reply:
Just to confirm my understanding here: the flag affects both the delimited proto and JSON versions of the execution log, right?